grails-app/services/org/pih/warehouse/core/DashboardService.groovy

```diff
@@ -573,6 +586,126 @@ class DashboardService {
         return reorderStock
     }
 
+    List<ReorderReportItemDto> getReorderReport(ReorderReportFilterCommand command) {
+        // Find inventory levels that don't have bin location set and have AT LEAST one of min/max qty set
+        List<InventoryLevel> inventoryLevels = InventoryLevel.createCriteria().list {
+            eq("inventory", AuthService.currentLocation.inventory)
```

Comment:
I thinkt it would be worth to have it in params (command), and if not specified then a fallback to current location. So user can technically request a report for another location without manually changing it.